src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -73,7 +75,9 @@ public static void configCheck(final ConfigType type) {
 			config.addDefault("Chat.Prefixes.Nonbender", "[Nonbender]");
 
 			config.addDefault("Board.Title", "&lAbilities");
-			config.addDefault("Board.SelectionPrefix", ">  &r");
+			config.addDefault("Board.Prefix.Text", "> ");
+			config.addDefault("Board.Prefix.SelectedColor", ChatColor.WHITE.toString());

```

Comment:
It would be cute to have the selected color be the color of their element.

Reply:
I have to agree, might be worth adding another option or having the string be `"ELEMENT"` (case-insensitive) or something. What are your thoughts @Simplicitee?